app/Http/Controllers/UserController.php

```diff
@@ -61,5 +61,43 @@ public function updateEmail(Request $request)
         // Redirect to route
         return redirect()->route('userdata');
     }
-    //TODO updatePassword 
+    public function changePassword(Request $request){
```

Comment:
Isn't there a Laravel built-in for this?

Reply:
> Shouln't be there also the personal information that the user filled out when registering?

Yes, I am working on it. I was thinking about showing the phone number, neptun code, and the faculties and workshops. But `{{$user->neptun}}` and `{{$user->phone_number}}` is not working as expected as with  `{{$user->name}}`. Do you have any idea why?

Reply:
> Isn't there a Laravel built-in for this?

As I looked up on this a little, I think there's only password reset built-in (with email verification etc...)